src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        String line;
+        Task[] list = new Task[100];
+        int noOfTask = 0;
+        Scanner in = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
         System.out.println("Hello from\n" + logo);
+
+        //Get input from user
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        line = in.nextLine();
```

Comment:
You may consider renaming the 'line' string variable into something like 'userInput' so that it is clearer that this string takes in user inputs.

Reply:
Yup! Sounds good